lib/main.dart

```diff
@@ -1,125 +1,141 @@
 import 'package:flutter/material.dart';
 
+import 'custom_widgets/middle_text.dart';
+import 'custom_widgets/title_widget.dart';
+import 'entities/Movie.dart';
```

Comment:
check why shit import is showing Movie as upper case, should be lower case, maybe the IDE is taking it either way